backend-api/src/functions/proxy/proxyHandler.ts

```diff
@@ -0,0 +1,142 @@
+import {
+  APIGatewayProxyEvent,
+  APIGatewayProxyEventHeaders,
+  APIGatewayProxyResult,
+  Context,
+} from "aws-lambda";
+
+import { dependencies, IMockProxyDependencies } from "./handlerDependencies";
+import { ConfigService } from "./configService/configService";
+
+export type StandardisedHeaders = {
+  [key in string]: string | number | boolean;
+};
+
+export async function lambdaHandlerConstructor(
+  dependencies: IMockProxyDependencies,
+  event: APIGatewayProxyEvent,
+  context: Context,
+): Promise<APIGatewayProxyResult> {
+  const logger = dependencies.logger();
+  logger.addContext(context);
+
+  const configService = new ConfigService();
+  const configResult = configService.getConfig(dependencies.env);
+  logger.log("STARTED");
+  if (configResult.isError) {
+    logger.log("ENVIRONMENT_VARIABLE_MISSING", {
+      errorMessage: configResult.value.errorMessage,
+    });
+    return {
+      headers: {
+        "Content-Type": "application/json",
+      },
+      statusCode: 500,
```

Comment:
As we return 500 more than once is it worth consting the response and then return something like `serverError500Response()`